hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/OmMetadataGenerator.java

```diff
@@ -157,6 +167,9 @@ enum Operation {
   private Operation[] operations;
   private boolean mixedOperation = false;
 
+  private final ReplicationConfig ratisThree =
```

Comment:
`ratisThree` is new config added for CREATE_KEY, CREATE_STREAM_KEY,CREATE_FILE_CREATE_STREAM_FILE. 
as, replicationConfig with default RATIS_THREE was configurable, but this new parameter is not configurable. IMO, either can use replicationConfig as same is already supported OR need support configuration for same for test.

Reply:
The redundant `ReplicationConfig` is removed and there is now only one configurable `ReplicationConfig`.